components/mediation/data-publishers/org.wso2.micro.integrator.observability/src/main/java/org/wso2/micro/integrator/obsrvability/handler/metrics/publisher/prometheus/reporter/PrometheusMetricCreator.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.micro.integrator.obsrvability.handler.metrics.publisher.prometheus.reporter;
+
+import org.apache.synapse.SynapseConstants;
+import org.wso2.micro.integrator.obsrvability.handler.metrics.publisher.prometheus.reporter.PrometheusReporter;
+import org.wso2.micro.integrator.obsrvability.handler.util.MetricConstants;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Class for creating Prometheus Metrics from the data scraped in the Metric Handler.
+ */
+public class PrometheusMetricCreator {
+
+    private static final String SERVICE_NAME = "service_name";
```

Comment:
Shall we move these constants to a single place and unify them

Reply:
Moved to MetricConstants.